lib/shark_client.js

```diff
@@ -311,8 +324,11 @@ SharkClient.prototype.put = function put(opts, cb) {
     assert.object(opts, 'options');
     assert.optionalNumber(opts.contentLength, 'options.contentLength');
     assert.string(opts.contentType, 'options.contentType');
-    assert.string(opts.objectId, 'options.objectId');
-    assert.string(opts.owner, 'options.owner');
+    assert.uuid(opts.bucketId, 'options.bucketId');
```

Comment:
Perhaps drop the "Options needs: ..." section in the block comment for this function as it is out of date and the asserts self-document it better.